src/nvim/normal.c

```diff
@@ -4361,7 +4361,7 @@ static void nv_ident(cmdarg_T *cap)
     /* put pattern in search history */
     init_history();
     add_to_history(HIST_SEARCH, buf, true, NUL);
-    normal_search(cap, cmdchar == '*' ? '/' : '?', buf, 0);
+    (void)normal_search(cap, cmdchar == '*' ? '/' : '?', buf, 0);
```

Comment:
Reviewers: is this still done nowadays, casting an unused return value to `void`? I haven't been writing any C lately.